lang/c++/api/Stream.hh

```diff
@@ -160,23 +160,23 @@ AVRO_DECL OutputStreamPtr memoryOutputStream(size_t chunkSize = 4 * 1024);
  * It does not copy the data, the byte array should remain valid
  * until the InputStream is used.
  */
-AVRO_DECL InputStreamPtr memoryInputStream(const uint8_t* data, size_t len);
+AVRO_DECL InputStreamPtr memoryInputStream(const uint8_t *data, size_t len);
 
 /**
  * Returns a new InputStream with the contents written into an
- * outputstream. The output stream must have been returned by
+ * output stream. The output stream must have been returned by
```

Comment:
`OutputStream` seems like the better fix.  Rewording this to be Returns a new InputStream that reads the contents written to `source` might be even better.

Reply:
Done